development-guidelines.md

```diff
@@ -61,27 +61,9 @@
    ЧТО поменялось, но и ГДЕ. Очевидные части пути (например .lua в названии
    файла) и точку в конце строки стоит опускать.
 
-   Примеры из lua-aplicado.
-   Плохо:
+   Приветствуется использование [тэгов][tagging].
 
-        git_is_directory_dirty added.
-        luarocks_read,
-        missing exports fix
-        minor code clean-up
-
-   Удовлетворительно:
-
-        shell/git.lua: git_is_directory_dirty added.
-        lua-aplicado/shell/luarocks.lua: luarocks_read,
-        lua-aplicado/shell/luarocks.lua: missing exports fix
-        lua-aplicado/shell/luarocks.lua: minor code clean-up
-
-   Хорошо:
-
-        shell/git: git_is_directory_dirty added
```

Comment:
Have no objections about examples. There are some already in `tag-guidelines.md`:

    [.^][readme] Reworded some lines.
    [?+][string-concatter] Added experimental garbage-collection heuristic.
    [!*] Debugged! (At least hope so.) Just want to mark this state...
    [/] [string/from_file] renamed to [file/as_string].

By the way I'd recommend to place space after digest tag:
`[.^] [readme] Reworded some lines.`
With this space message is more structured and seekable.